capstone-frontend/src/components/ListPage.jsx

```diff
@@ -23,7 +26,7 @@ class ListPage extends Component {
     const gbookIDs = bookList[0].gbookIDs;
 
     if (!gbookIDs.length) {
-      this.setState({ loading: false, empty: true });
+      this.setState({ loading: false, empty: true, bookList: bookList[0] });
```

Comment:
I know this isn't technically part of the change you're making here, but the constant `bookList` being assigned to the API response should either be called `bookLists` plural since it's a collection, or should be assigned to the first entry in the array (e.g. `.then(resp => resp.json()).then(bookLists => bookLists[0])`). As it is now this line looks a little confusing.